### ISSUE TITLE
Missed messages in a group chat

### ISSUE THREAD

**Issue by saledjenic** [status-im/status-desktop#11383]
After I run the app after ~10 days, I left the app for some time ~10 mins (to receive all missed messages) and started reading the WalletTeam group chat. 

Then I was hearing sounds like ne messages were receiving but there were no new messages (new like received after I started the app), but those sounds were played for newly received old messages. This can be the first issue?!?! Do we need to play a new message sound for history messages?

Then I realized, just because I saw a lot of reply messages where an original message, a group member replied to, was not displayed, that I am missing some messages and because of that I couldn't understand the whole context a group members were discussing. This is definitely an issue worth checking, cause the thing could be even worse if nobody replied to those messages, then I couldn't even know that I don't have all messages.

**Comment by jrainville** [comment]
> After I run the app after ~10 days, I left the app for some time ~10 mins (to receive all missed messages) and started reading the WalletTeam group chat.

I didn't understand that part when reading the message for the first time. 

This issue is normal then, we only fetch message for the last 9 days (might be 7 even) cc @richard-ramos to confirm

**Comment by saledjenic** [comment]
@jrainville don't fetch messages for the last 30 days?

**Comment by jrainville** [comment]
> @jrainville don't fetch messages for the last 30 days?

No. we store 30 days of messages, but only fetch 7-9, otherwise it would be a very big/long request.

I know Richard split the fetching per day, so maybe we'll expand to more days.

**Comment by jrainville** [comment]
Gonna close this issue since it's "per design". If we decide to increase the fetch range, then it will fix this also.

**Comment by jrainville** [comment]
@Cuteivist @saledjenic did you see if the notification was from a group message (maybe a group you couldn't see), or it's just that it's a notification in a channel where you had already read recent messages and thus there was no unread marker?
